spatial-tracking-explainer.md

```diff
@@ -148,20 +148,20 @@ function onSessionStarted(session) {
 ```
 
 ### Viewer reference spaces
-A _viewer_ reference space tracks the origin of the viewer. This type of reference space is primarily used for creating inline experiences with no tracking information. Instead, developers may use `XRReferenceSpace.originOffset` which is described in the [Application supplied transforms section](#application-supplied-transforms). An example usage of an _viewer_ reference space is a furniture viewer that will use [click-and-drag controls](#click-and-drag-view-controls) to rotate the furniture. It also supports cases where the developer wishes to avoid displaying any type of tracking consent prompt to the user prior while displaying inline content.
+A _viewer_ reference space's origin is always at the position and orientation of the viewer device. This type of reference space is primarily used for creating inline experiences with no tracking of the viewer relative to it's physical environment. Instead, developers may use `XRReferenceSpace.originOffset` which is described in the [Application supplied transforms section](#application-supplied-transforms). An example usage of an _viewer_ reference space is a furniture viewer that will use [click-and-drag controls](#click-and-drag-view-controls) to rotate the furniture. It also supports cases where the developer wishes to avoid displaying any type of tracking consent prompt to the user prior while displaying inline content.
```

Comment:
>`XRReferenceSpace.originOffset`

Are we planning to fixing the explainer text about originOffset in a separate PR?

Reply:
Yeah, that's a large enough change that I'd feel more comfortable handling it separately from this issue.

Reply:
Fine by me.  Can you file an issue to track that?

Reply:
#634